src/components/ec-alert/ec-alert.spec.js

```diff
@@ -0,0 +1,114 @@
+import Vue from 'vue';
+import { mount } from '@vue/test-utils';
+import EcAlert from './ec-alert.vue';
+import { withMockedConsole } from '../../../tests/utils/console';
+
+describe('EcAlert', () => {
+  function mountAlert(props, mountOpts) {
+    return mount(EcAlert, {
+      propsData: {
+        title: 'Title example',
+        type: 'info',
+        ...props,
+      },
+      ...mountOpts,
+    });
+  }
+
+  it('should throw if no props were given', () => {
+    withMockedConsole((errorSpy) => {
+      mount(EcAlert);
+      expect(errorSpy).toHaveBeenCalledTimes(2);
+      expect(errorSpy.mock.calls[0][0]).toContain('Missing required prop: "type"');
+      expect(errorSpy.mock.calls[1][0]).toContain('Missing required prop: "title"');
+    });
+  });
+
+  it('should display only with a title given and the type', () => {
+    const wrapper = mountAlert({ title: 'Random Title', type: 'info' });
+
+    expect(wrapper.element).toMatchSnapshot();
+  });
+
+  it('should display with a subtitle given', () => {
+    const wrapper = mountAlert({ subtitle: 'Subtitle example' });
+    expect(wrapper.element).toMatchSnapshot();
+  });
+
+  it('should display with a button with the text given in the buttonText', () => {
+    const wrapper = mountAlert({ buttonText: 'Button text' });
+
+    expect(wrapper.element).toMatchSnapshot();
+  });
+
+  it('should use the given type for the alert type and for the button too', () => {
+    const wrapper = mountAlert({ type: 'error', buttonText: 'Warning button' });
+
+    expect(wrapper.element).toMatchSnapshot();
+  });
+
+  it.each(['error', 'info', 'success', 'warning'])('should use the type "%s"', (type) => {
```

Comment:
lovely use of array :heart_eyes: 

we can probably get rid of parenthesis in `type`

are we passing formatter/linter to the specs?

Reply:
![image](https://user-images.githubusercontent.com/29793161/68664415-49a3cb00-0538-11ea-965b-d66a8e7e3082.png)
This is without parenthesis, I think we cannot skip that.
The error is _Expected parentheses around arrow function argument having a body with curly braces.eslint(arrow-parens)_

Reply:
oh, I did't catch that, my :eyes: went directly to the argument, thanks!